### PR TITLE
Bug 1862508: Create PVC dropdown text update

### PR DESCRIPTION
**Pull request by glekner**

### CODE REVIEW

**Comment by glekner** [comment]
@matthewcarleton @yfrimanm wdyt?

File: frontend/packages/kubevirt-plugin/src/plugin.tsx [hunk 1]
```diff
@@ -494,7 +494,7 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'PVCCreateProp',
     properties: {
-      label: 'New with Data Upload',
+      label: 'With Data upload form',
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/plugin.tsx:497]: the BZ say "With form and Data upload" ?

Comment (glekner) [reply frontend/packages/kubevirt-plugin/src/plugin.tsx:497]: yes and I'm asking from Matt to rethink it :)

**Pull request merged**